Automattic/woocommerce-payments
# E2E - Migrate dev tools flows to use WP-CLI

Fixes #4402 

#### Changes proposed in this Pull Request

Previously, enabling features via dev tools was done like a regular test flow, which is not ideal since dev tools features are used only for development. This PR adds updates the dev tools flows to use WP-CLI

#### Testing instructions

* Confirm the E2E tests on this PR is passing as expected.
* Confirm that the test `shopper-myaccount-save-card-and-checkout-upe.spec.js` under `WCPay - Shopper` is passing since the test uses the new WPCLI flows.
* Performance tests can be tested locally by running `npm run test:e2e-performance`
-------------------


**Post merge**
